### ISSUE TITLE
Do you  have infoNCE loss function cpde

### ISSUE THREAD

**Issue by jiangxinke** [RElbers/info-nce-pytorch#5]

**Comment by RElbers** [comment]
The code is in the __init__.py file.
